static/css/restyle.less

```diff
@@ -910,7 +910,14 @@ button.good {
   .category.more-info,
   .vital a,
   #reviews-link {
-    opacity: 0.4;
+    opacity: .8;
```

Comment:
Actually this might be better as rgba rather than using opacity on the whole thing. Moot point if we just make it match the headings though.